lib/aquifer.api.js

```diff
@@ -154,6 +184,95 @@ class Aquifer {
       resolve();
     })
   }
+
+  /**
+   * Get initial project configuration.
+   * @returns {object} promise object.
+   */
+  getConfig() {
+    return new Promise((resolve, reject) => {
+      this.initialConfig = {};
+
+      // If this project is initialized, load the JSON path.
+      if (this.initialized) {
+        // Calculate paths to json, and source directory.
+        let jsonPath = path.join(this.projectDir, 'aquifer.json');
+        let localJsonPath = path.join(this.projectDir, 'aquifer.local.json');
+
+        this.initialConfig = jsonFile.readFileSync(jsonPath);
+
+        // Extend config with aquifer.local.json.
+        if (fs.existsSync(localJsonPath)) {
+          this.initialConfig = _.defaultsDeep(jsonFile.readFileSync(localJsonPath), this.initialConfig);
+        }
+      }
+
+      resolve();
+    })
+  }
+
+  /**
+   * Redirects execution to the proper version of aquifer.
+   * @returns {object} promise object.
+   */
+  switchVersion() {
+    return new Promise((resolve, reject) => {
+      // No need to switch aquifer versions if we don't have an aquifer project, yet.
+      if (!this.initialized) {
+        resolve();
+        return;
+      }
+
+      if (!this.initialConfig.hasOwnProperty('version')) {
+        this.console.log('You have not specified an aquifer version in aquifer.json. Using the default installed version: ' + this.version, 'warning');
+        resolve();
+        return;
+      }
+
+      if (this.initialConfig.version === this.version) {
+        resolve();
+        return;
+      }
```

Comment:
techincally... consiser combining this and the first If... they end in the same result.